home.css

```diff
@@ -0,0 +1,106 @@
+.adv-container {
+    display: flex;
+    justify-content: center;
+    font-style: italic;
+}
+
+.adv-container img {
+    margin: auto;
+    width: 80vw;
+    height: 30vw;

```

Comment:
Suggestion: for height we can use `vh` instead of `vw` as `vw`  takes 1% of viewport width and `vh` takes 1% of viewport height, correct me if I am wrong

Reply:
yes  That's right. I accidently set in vw for height.